apps/system/js/software_button_manager.js

```diff
@@ -332,10 +332,10 @@
           var isPortrait = this._currentOrientation.contains('portrait');
           var newOrientation = Service.query('fetchCurrentOrientation');
           if (isPortrait && newOrientation.contains('landscape')) {
-            this.element.style.right = this.element.style.bottom;
+            this.element.style.right = 0;
```

Comment:
While debugging I noticed that the `this.element.style.bottom` value was `""` instead of `0`. I first tried setting the `bottom: 0;` style via the css file so this code change wouldn't be needed, but that didn't seem to work.